src/validators.js

```diff
@@ -10,14 +10,14 @@ const removeNativeKeys = (req, res, next) => {
 	next();
 }
 
-// size of the JSON body should be larger than 100KB
+// validator: size of payload < 100KB
 const sizeValidator = (req, res, next) => {
 	if (req.method === 'POST' || req.method === 'PUT') {
 		if (Object.keys(req.body).length > 0) {
 			const memorySize = helper.memorySizeOf(req.body);
 			req['bodySize'] = memorySize;
-			// memorySize is size in bytes
-			if (memorySize > 100000) {
+			// memorySize is size in bytes => 1024 * 1024
+			if (memorySize > (1024 * 1024)) {
```

Comment:
@iambenkay sorry I didn't note this while reviewing it first. 100KB = 100 * 1024